ifrs17/Import/ImportStorage.ipynb

```diff
@@ -308,8 +309,10 @@
                 "\n    }",
                 "\n    ",
                 "\n    public YieldCurve GetYieldCurve(ImportIdentity id, string economicBasis) => (economicBasis, GetYieldCurvePeriod(id)) switch {",
-                "\n            (EconomicBases.C, PeriodType.BeginningOfPeriod) => CurrentYieldCurve[id.DataNode][PreviousPeriod],",
-                "\n            (EconomicBases.C, PeriodType.EndOfPeriod) => CurrentYieldCurve[id.DataNode][CurrentPeriod],",
+                "\n            (EconomicBases.C, PeriodType.BeginningOfPeriod ) => GetShift(id.ProjectionPeriod) > 0 ",
```

Comment:
instead of 0 let's use MonthInAYear const (>=)

Reply:
Done.